lua/entities/ent_jack_generator/cl_init.lua

```diff
@@ -43,74 +43,77 @@ function ENT:Draw()
     local Up = self:GetUp()
     local Forward = self:GetForward()
     local Ang2 = self:GetAngles()
-    Ang:RotateAroundAxis( Ang:Forward(), self.RotateAngle )
-    self.RotateAngle = self.RotateAngle + self.RotSpeed
 
-    if self.RotateAngle > 360 then
-        self.RotateAngle = 0
+    local selfTbl = self:GetTable()

```

Comment:
Micro optimization - localizing `Engine1`, `Engine2`, etc. should be a fairly good improvement too